src/components/sde/tests/Simple2/simple2.hpp

```diff
@@ -0,0 +1,19 @@
+#if !defined(SIMP2_LIB_H)
+#define SIMP2_LIB_H
+
+#include "sde_lib.h"
+#include "sde_lib.hpp"
+
+class Simple{
+    private:
```

Comment:
Same suggestion as with the class located in `Minimal_Test++.cpp`:

- `public` before `private`.
- `public` and `private` are not indented.